source/make-an-offer.html.md.erb

```diff
@@ -15,18 +17,24 @@ POST /applications/:id/offer
 POST /applications/11fc0d3b2f/offer
 ```
 
-Request body:
+Request body for an **unconditional** offer:
+
+```json
+<%= unconditional_offer_creation_json %>
+```
+
+Request body for a **conditional** offer:
```

Comment:
Perhaps we can spell out what is different about a conditional offer: 

```suggestion
Add `conditions` to the request body for a **conditional** offer:
```

Reply:
Makes sense! I've updated this now.